python-lib/dku_data_processing/format.py

```diff
@@ -17,7 +13,12 @@ def normalize(x):
     """
     Will normalize data to have 75% of the values between 0 and 1
     """
-    x_scaled = (x - np.percentile(x, 12.5)) / (np.percentile(x, 87.5) - np.percentile(x, 12.5))
+    perc_left = np.percentile(x, 12.5)
+    perc_right = np.percentile(x, 87.5)
```

Comment:
To me, `right_percentile` and `left_percentile` would sound more meaningful than `perc_left` and `perc_right`